core/src/test/scala/unit/kafka/admin/TopicCommandWithAdminClientTest.scala

```diff
@@ -673,10 +678,14 @@ class TopicCommandWithAdminClientTest extends KafkaServerTestHarness with Loggin
       Collections.singletonList(new NewTopic(testTopicName, partitions, replicationFactor).configs(configMap))).all().get()
     waitForTopicCreated(testTopicName)
     TestUtils.generateAndProduceMessages(servers, testTopicName, numMessages = 10, acks = -1)
+    Thread.sleep(10)
```

Comment:
Do we want to leave a comment here to say that we are trying to ensure that messages are produced in multiple batches? I do wonder if there's a better way to achieve this via a producer setting, e.g. `batch.size`?

Reply:
We can call `flush` on the producer to force it to send the messages.

Reply:
Looking a bit more, we actually close the producer after we generate the messages, so I don't think this sleep is needed at all.